webapp/channels/src/components/integrations/installed_incoming_webhook.tsx

```diff
@@ -151,34 +191,29 @@ export default class InstalledIncomingWebhook extends React.PureComponent<Props>
                     {description}
                     <div className='item-details__row'>
                         <span className='item-details__url word-break--all'>
-                            <FormattedMessage
-                                id='installed_integrations.url'
-                                defaultMessage='URL: {url}'
-                                values={{
-                                    url: incomingWebhookId,
-                                }}
+                            {incomingWebhookId}
+                            <CopyText
+                                value={incomingWebhookIdValue}
                             />
-                            <span>
-                                <CopyText
-                                    value={incomingWebhookId}
-                                />
-                            </span>
                         </span>
                     </div>
                     <div className='item-details__row'>
-                        <span className='item-details__creation'>
-                            <FormattedMessage
-                                id='installed_integrations.creation'
-                                defaultMessage='Created by {creator} on {createAt, date, full}'
-                                values={{
-                                    creator: this.props.creator.username,
-                                    createAt: incomingWebhook.create_at,
-                                }}
+                        <span className='item-details__channel_name word-break--all'>
+                            {channelDisplayName}
+                            <CopyText
+                                value={channel.display_name ?? 'N/A'}
```

Comment:
please, change this to have translation too.

Reply:
I have use the channelDisplayName as the value for CopyText and  channelDisplayName is already have translation.

Reply:
Hi @pvev . Please check the changes I have made. And thank you for your thorough feedbacks and reviews.